Framework/PythonInterface/plugins/algorithms/LRAutoReduction.py

```diff
@@ -425,7 +426,7 @@ def _get_direct_beam(self, meta_data_run, data_set):
         direct_beam_runs_str = self._read_property(meta_data_run, "direct_beam_runs",
                                                    _direct_beam_runs, is_string=True)
         # The direct runs in the DAS logs are stored as a string
-        if isinstance(direct_beam_runs_str, (str, unicode)):
+        if isinstance(direct_beam_runs_str, str):
```

Comment:
No it doesn't. `isinstance(u"foobar",(str,unicode)) = True` while `isinstance(u"foobar",str) = False`. But `unicode` isn't a type in python 3 so something else is needed here.

How about `isinstance(u"foobar",(str,type(u'')))`? It works for python 2 and 3. Unless there is a better way?